rustup-init.sh

```diff
@@ -1,9 +1,13 @@
-#!/bin/bash
+#!/bin/sh
+# shellcheck shell=dash
 
 # This is just a little script that can be downloaded from the internet to
 # install rustup. It just does platform detection, downloads the installer
 # and runs it.
 
+# It runs on Unix shells like {a,ba,da,k,z}sh. It uses the common `local`
+# extension. Note: Most shells limit `local` to 1 var per line, contra bash.
```

Comment:
```suggestion
# extension. Note: Most shells limit `local` to 1 var per line but bash.
```

Reply:
The intent of the line is to warn a future dev against testing `local`'s behavior with bash and expecting it to be the same across shells.

Rewording would be fine if the turn of phrase is considered too unusual, but this particular diff would not convey my intent because "but" does not carry quite the same meaning of "in opposition to", and offers a misleading "every shell but bash" interpretation. It's possible but doubtful that bash is the only one that supports a generic extension of multiple locals in 1 line.

Reply:
I agree with @workingjubilee here.  There may be a nicer way to put it, but your suggestion definitely isn't it.  I'm going to merge as is.